### PR TITLE
Update plugin-portal.mdx

### PR DESCRIPTION
**Pull request by acahn**
Adding in a few community supported plugins

### CODE REVIEW

File: website/content/docs/plugin-portal.mdx [hunk 1]
```diff
@@ -122,11 +122,13 @@ Plugin authors who wish to have their plugins listed may file a submission via a
 
 ### Auth
 
+- [Github Actions](https://github.com/Mongey/vault-plugin-auth-github-actions)
```
Comment (calvn) [inline website/content/docs/plugin-portal.mdx:125]: How does this compare to `vault-action`? At a glance, it seems that it's an auth method that's trading in a GH token (along with some other data from the run), but it's also able to fetch secrets from vault once authenticated which is a bit non-standard for an auth method to do.

Comment (narayan-iyengar) [reply website/content/docs/plugin-portal.mdx:125]: I believe the `vault-action` auth creds are stored either in the repo or environment variables.
the plugin listed here is a different auth mechanism and uses the assigned GITHUB_TOKEN as a source of identity as opposed to app role or a personal access token which can be long lived

Comment (narayan-iyengar) [reply website/content/docs/plugin-portal.mdx:125]: to add...maybe the overlap here is the fetching secrets part?

Comment (acahn) [reply website/content/docs/plugin-portal.mdx]: @calvn, I've removed the plugin from the PR.  We can chat about it later if we still want to add it in.

**Review by hsimon-hashicorp — APPROVED** [review]

**Pull request closed without merging**